src/Apps/Order/Components/ArtworkSummaryItem.tsx

```diff
@@ -27,15 +27,12 @@ const ArtworkSummaryItem: React.SFC<ArtworkSummaryItemProps> = ({
 }) => {
   const artwork = get({}, props => lineItems.edges[0].node.artwork)
 
-  const {
-    artist_names,
-    title,
-    date,
-    shippingOrigin,
-    image: {
-      resized_ArtworkSummaryItem: { url: imageURL },
-    },
-  } = artwork
+  const { artist_names, title, date, shippingOrigin, image } = artwork
+
+  const imageURL =
+    image &&
```

Comment:
#minor but for next time, we have a custom `get` helper which allows us to safely access properties!